### ISSUE TITLE
Issue querying temperature data using readWQPqw

### ISSUE THREAD

**Issue by cander87** [DOI-USGS/dataRetrieval#669]
**What is your question?**
Why am I only getting a subset of data available from a site when querying temperature data using readWQPqw?

I.e., I'm trying to query all temperature data for USGS-023060013 up until 05.22.23 (today), but I am only getting 4 data points using the code below. 

**To Reproduce**
If possible, narrow down the question to a `dataRetrieval` query:

```r
water_temp_hillsborough_023060013 <- readWQPqw("USGS-023060013",   # Sites to query
                                               "00010",            # Parameter to query
                                               "",                 # Beginning of time range
                                               "2023-05-22")       # End of time range
```

**Expected behavior**
I expect this code to query all raw water temperature data in the period of record for station USGS-023060013. 

**Screenshots**
![image](https://github.com/DOI-USGS/dataRetrieval/assets/133255523/1f40079e-be2b-49c7-a8da-78a8e1f80dfa)

**Session Info**
Please include your session info:
R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] lubridate_1.9.2      forcats_1.0.0        stringr_1.5.0        dplyr_1.1.2         
 [5] purrr_1.0.1          readr_2.1.4          tidyr_1.3.0          tibble_3.2.1        
 [9] ggplot2_3.4.2        tidyverse_2.0.0      dataRetrieval_2.7.12

loaded via a namespace (and not attached):
 [1] bit_4.0.5        gtable_0.3.3     compiler_4.3.0   crayon_1.5.2     tidyselect_1.2.0
 [6] parallel_4.3.0   scales_1.2.1     R6_2.5.1         generics_0.1.3   curl_5.0.0      
[11] munsell_0.5.0    pillar_1.9.0     tzdb_0.3.0       rlang_1.1.1      utf8_1.2.3      
[16] stringi_1.7.12   bit64_4.0.5      timechange_0.2.0 cli_3.6.1        withr_2.5.0     
[21] magrittr_2.0.3   grid_4.3.0       vroom_1.6.3      rstudioapi_0.14  hms_1.1.3       
[26] lifecycle_1.0.3  vctrs_0.6.2      glue_1.6.2       fansi_1.0.4      colorspace_2.1-0
[31] httr_1.4.6       tools_4.3.0      pkgconfig_2.0.3 

**Additional context**

**Comment by ldecicco-USGS** [comment]
Since this is a USGS site, you can use the function "whatNWISdata" to see the available data. You'll want to then look at the data_type_cd to see where to get the available data. "qw" data is available from the Water Quality Portal, but "dv" or "uv" are only available from NWIS services ("dv" and "uv" respectively).

```r
what_data <- whatNWISdata(siteNumber = "023060013")

library(dplyr)
what_data |> 
  filter(parm_cd == "00010") |> 
  select(data_type_cd, stat_cd, 
         loc_web_ds,
         begin_date, end_date, count_nu)

  data_type_cd stat_cd    loc_web_ds begin_date   end_date count_nu
1           dv   00001    TOP, [TOP] 1999-09-11 2023-05-21     5453
2           dv   00001 BOT, [BOTTOM] 1999-09-11 2023-05-21     5411
3           dv   00002    TOP, [TOP] 1999-09-11 2023-05-21     5443
4           dv   00002 BOT, [BOTTOM] 1999-09-11 2023-05-21     5408
5           dv   00003    TOP, [TOP] 1999-10-01 2023-05-21     4196
6           dv   00003 BOT, [BOTTOM] 1999-10-01 2023-05-21     4153
7           qw    <NA>          <NA> 2011-06-29 2011-10-03        4
8           uv    <NA>           TOP 2011-06-30 2023-05-22     4344
9           uv    <NA>           BOT 2011-06-30 2023-05-22     4344

```

The "qw" data is what you get from WQP. It seems correct to see 4 data points returned.

To get that "dv" (daily) data, you would ask for it like this. The "stat_cd"s available (based on the table above) are "00001" (max), "00002" (min), "00003" (mean):

```r
dv <- readNWISdv("023060013", 
                 parameterCd = "00010",
                 statCd = c("00001", "00002", "00003"))
```

To get the "uv" data (instantaneous, continuous):

```r
uv <- readNWISuv("023060013", 
                 parameterCd = "00010")
```

In summary....you need to check the "data_type_cd" to find what data service to use. If it's "qw", use the Water Quality Portal (that is just for discrete data). If it's anything else, it'll be in NWIS, and you'll need to use the appropriate NWIS service (dv, uv, gwl, etc.) Let me know if any of that isn't clear.

**Comment by cander87** [comment]
That makes a lot of sense! Thank you!

Chris
